File: packages/vscode-extension/tests/fixtures/textmate/simple-match.java
String a = """//cypher
    // This is a comment
    MATCH (n:Label) RETURN function(n.property)
"""

String b = """/* cypher */
    // This is a comment
    MATCH (n:Label) RETURN function(n.property)
"""

String c = "/*cypher*/ MATCH (n:Label) RETURN function(n.property)"

// This one shouldn't highglight
String d = "//cypher MATCH (n:Label) RETURN function(n.property)"

File: packages/vscode-extension/tests/fixtures/textmate/simple-match.js

const a = /* cypher */`
    // This is a comment
    MATCH (n:Label) RETURN function(n.property)
`

const b = /*cypher*/`  
    // This is a comment
    MATCH (n:Label) RETURN function(n.property)
`

const c = `//cypher
    // This is a comment
    MATCH (n:Label) RETURN function(n.property)
`

const e = `/* cypher */
    // This is a comment
    MATCH (n:Label) RETURN function(n.property)
`

const f = `/*cypher*/
    // This is a comment
    MATCH (n:Label) RETURN function(n.property)
`

const g = "/*cypher*/ MATCH (n:Label) RETURN function(n.property)"

const h = '/*cypher*/ MATCH (n:Label) RETURN function(n.property)'

// This one shouldn't highglight
const i = "//cypher MATCH (n:Label) RETURN function(n.property)"

// This one shouldn't highglight
const j = '//cypher MATCH (n:Label) RETURN function(n.property)'